linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faildelay_delay/ansible/ubuntu.yml

```diff
@@ -0,0 +1,12 @@
+# platform = multi_platform_ubuntu
```

Comment:
Please, rename this file to `shared.yml`. This remediation might be useful for other products too.

Reply:
please also consolidate the bash remediation into a single `shared.sh`